Sources/Rendering/Core/ColorTransferFunction/index.js

```diff
@@ -1030,7 +1033,10 @@ function vtkColorTransferFunction(publicAPI, model) {
       rgba[3] = 1.0; // NodeColor is RGB-only.
       return;
     }
-    publicAPI.getNanColor(rgba);
+    const nanColor = publicAPI.getNanColor();
```

Comment:
does that call do a new allocation? Maybe use getNanColorAsReference() if available instead?

Reply:
Ah good point. Yes, I think there's a copy here. I forgot about the AsReference methods.